mindsdb/integrations/handlers/writer_handler/requirements.txt

```diff
@@ -1,14 +1 @@
-requests>=2.28.1
-charset-normalizer>=2.1.1
-protobuf<3.21.0 # for lightwood
-SQLAlchemy
-chromadb~=0.4.8
-duckdb==0.7.1
-pydantic>=1.10.8
-sentence_transformers==2.2.2
-markupsafe == 2.0.1
-faiss-cpu>=1.7.4 #todo check support for 'faiss-gpu'
-datasets>=2.14.4
-nltk>=3.8.1
-rouge-score>=0.1.2
-dill==0.3.6 #for lightwood
+-r ../rag_handler/requirements.txt # same requirements
```

Comment:
This will work with `pip install` right? do we need the -r or it can be used from `pip install -r ,,,`>

Reply:
yep it works, I tested it out locall.y

the idea is if user `pip -r ../writer_handler/requirements.txt`, it will install requirments from `rag_handler` dir.

As there is a massive overlap, it figured it is easier to maintain a single list of requirements

Reply:
-r ,,,